README.md

```diff
@@ -247,15 +247,24 @@ Returned promise rejects with an error if there is no previously authenticated u
 }
 ```
 
-# Build Setup
+# Examples
+
+Before checking examples, please copy `examples/.env.example` to `examples/.env` and add the correct AWS crendentials to it.
 
 ``` bash
+cd examples
+
 # install dependencies
 npm install
 
 # serve examples with hot reload at localhost:8080
+npm run server # start API server for UUID examples
 npm run dev
```

Comment:
Maybe use node-foreman here? A single Procfile takes care of running both servers. There is already a Procfile in this `examples/` folder, just mention it in the readme and make it run both servers.

Probably also a good idea to mention that both of these services will be executed, and these will be using the `.env` file for configuration.